ui/package.json

```diff
@@ -40,7 +40,7 @@
     "@fortawesome/vue-fontawesome": "^3.0.0-4",
     "@vue-js-cron/ant": "^1.1.3",
     "@vue-js-cron/core": "^3.7.1",
-    "ant-design-vue": "^3.2.9",
+    "ant-design-vue": "^3.2.20",
```

Comment:
Full testing of UI may be required, but I'm happy to drop this if advised cc @shwstppr @Pearl1594 @DaanHoogland and other reviewers